<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Paths</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;quvi:</string>
         </property>
         <property name="buddy">
          <cstring>quviPathEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="quviPathEdit">
         <property name="text">
          <string>&lt;replace with path and any additional options&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="browseQuviBtn">
         <property name="text">
          <string>&amp;Browse...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>c&amp;url:</string>
         </property>
         <property name="buddy">
          <cstring>curlPathEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="curlPathEdit">
         <property name="text">
          <string>&lt;replace with path and any additional options&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="browseCurlBtn">
         <property name="text">
          <string>B&amp;rowse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>P&amp;layer:</string>
         </property>
         <property name="buddy">
          <cstring>playerPathEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="playerPathEdit">
         <property name="text">
          <string>&lt;replace with path and any additional options&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="browsePlayerBtn">
         <property name="text">
          <string>Bro&amp;wse...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>u&amp;mph:</string>
         </property>
         <property name="buddy">
          <cstring>umphPathEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="umphPathEdit">
         <property name="text">
          <string>&lt;replace with path and any additional options&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="browseUmphBtn">
         <property name="text">
          <string>Brow&amp;se...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>S&amp;ave videos:</string>
         </property>
         <property name="buddy">
          <cstring>saveDirEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="saveDirEdit">
         <property name="text">
          <string>&lt;replace with video save directory path&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="browseSaveDirBtn">
         <property name="text">
          <string>Brows&amp;e...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>&amp;Filename</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>F&amp;ilename format:</string>
         </property>
         <property name="buddy">
          <cstring>filenameFormatEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="filenameFormatEdit">
         <property name="text">
          <string>%t.%s</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="filenameFormatCombo">
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Title, ID, Suffix</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Title, ID, Host, Suffix</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Tit&amp;le regexp:</string>
         </property>
         <property name="buddy">
          <cstring>regexpEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="regexpEdit">
         <property name="text">
          <string>/(\w|\s)/g</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="regexpCombo">
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Word characters only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Digit characters only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Non-word characters only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Non-digit characters only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Non-whitespace characters only</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Preview:</string>
         </property>
         <property name="buddy">
          <cstring>previewEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="previewEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="refreshBtn">
         <property name="text">
          <string>&amp;Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Appeara&amp;nce</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Program Icon</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QComboBox" name="iconCombo">
            <item>
             <property name="text">
              <string>Default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Custom</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="iconPathEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="browseIconBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Op&amp;tions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QCheckBox" name="minWhenStartsBox">
           <property name="text">
            <string>&amp;Minimize when download starts</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="minToTrayBox">
           <property name="text">
            <string>M&amp;inimize to system tray</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="playWhenDoneBox">
           <property name="text">
            <string>Pl&amp;ay video when done downloading</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="stayOnTopBox">
           <property name="text">
            <string>&amp;Stay on top</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="showReminderBox">
           <property name="text">
            <string>S&amp;how reminder when NomNom starts</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="dontPromptFilenameBox">
           <property name="text">
            <string>&amp;Do not prompt for filename</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="clearRecentBtn">
           <property name="text">
            <string>Cl&amp;ear recent</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="languageBtn">
           <property name="text">
            <string>&amp;Language...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Config:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="configEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>quviPathEdit</tabstop>
  <tabstop>browseQuviBtn</tabstop>
  <tabstop>curlPathEdit</tabstop>
  <tabstop>browseCurlBtn</tabstop>
  <tabstop>playerPathEdit</tabstop>
  <tabstop>browsePlayerBtn</tabstop>
  <tabstop>umphPathEdit</tabstop>
  <tabstop>browseUmphBtn</tabstop>
  <tabstop>saveDirEdit</tabstop>
  <tabstop>browseSaveDirBtn</tabstop>
  <tabstop>filenameFormatEdit</tabstop>
  <tabstop>filenameFormatCombo</tabstop>
  <tabstop>regexpEdit</tabstop>
  <tabstop>regexpCombo</tabstop>
  <tabstop>previewEdit</tabstop>
  <tabstop>refreshBtn</tabstop>
  <tabstop>iconCombo</tabstop>
  <tabstop>iconPathEdit</tabstop>
  <tabstop>browseIconBtn</tabstop>
  <tabstop>minWhenStartsBox</tabstop>
  <tabstop>minToTrayBox</tabstop>
  <tabstop>playWhenDoneBox</tabstop>
  <tabstop>stayOnTopBox</tabstop>
  <tabstop>showReminderBox</tabstop>
  <tabstop>dontPromptFilenameBox</tabstop>
  <tabstop>clearRecentBtn</tabstop>
  <tabstop>languageBtn</tabstop>
  <tabstop>configEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browsePlayerBtn</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>onBrowsePlayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseQuviBtn</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>onBrowseQuvi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearRecentBtn</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>onClearRecent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regexpCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Preferences</receiver>
   <slot>onRegexpChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filenameFormatCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Preferences</receiver>
   <slot>onFilenameFormatChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshBtn</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>onRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filenameFormatEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>Preferences</receiver>
   <slot>onFilenameFormatEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regexpEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>Preferences</receiver>
   <slot>onRegexpEditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseCurlBtn</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>onBrowseCurl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>languageBtn</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>onLanguage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseUmphBtn</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>onBrowseUmph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseSaveDirBtn</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>onBrowseSaveDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iconCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Preferences</receiver>
   <slot>onProgramIconChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseIconBtn</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>onBrowseProgramIcon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBrowseQuvi()</slot>
  <slot>onBrowsePlayer()</slot>
  <slot>onClearRecent()</slot>
  <slot>onBrowseUmph()</slot>
  <slot>onFilenameFormatChanged(int)</slot>
  <slot>onRegexpChanged(int)</slot>
  <slot>onRefresh()</slot>
  <slot>onFilenameFormatEditingFinished()</slot>
  <slot>onRegexpEditingFinished()</slot>
  <slot>onBrowseCurl()</slot>
  <slot>onBrowseSaveDir()</slot>
  <slot>onLanguage()</slot>
  <slot>onProgramIconChanged(int)</slot>
  <slot>onBrowseProgramIcon()</slot>
 </slots>
</ui>
